src/javascript/ContentManager/ContentManager.utils.js

```diff
@@ -59,6 +59,36 @@ function ellipsizeText(text, maxLength) {
     return ellipsize(text, maxLength || 100, {chars: [' ', '&']});
 }
 
+function getZipName(name) {
+    if (name.lastIndexOf('.') > 0) {
+        return name.substr(0, name.lastIndexOf('.')) + '.zip';
+    }
+
+    return name + '.zip';
+}
+
+function getNameWithoutExtension(name) {
+    if (name.lastIndexOf('.') > 0) {
+        return name.substr(0, name.lastIndexOf('.'));
+    }
+
+    return name;
+}
+
+function getNewCounter(nodes) {
+    let max = 0;
+    nodes.forEach(node => {
+        let counter = node.name.match(/[0-9]+.zip/g);
+        if (counter !== null) {
+            counter = counter[0].match(/[0-9]+/g);
+            if (counter !== null && counter[0] > max) {
+                max = counter;
+            }
+        }
+    });
+    return parseFloat(max) + 1;
```

Comment:
Why parseFloat?

Reply:
I used it only to force addition rather than concatenation